### ISSUE TITLE
ReportRepoSize not working after 3.x upgrade.

### ISSUE THREAD

**Issue by tahershaakir** [Autodesk/hubble#237]
We noticed that ReportRepoSize stopped working ( gives no detailed output ) after the upgrade to 3.x.
( https://github.com/Autodesk/hubble/blob/master/updater/reports/ReportRepoSize.py) 

```Started update of repository-size.tsv

==== /tmp/hubble-data/repository-size.tsv ======================================
date	repositories >1GB
2022-01-04	0

================================================================================
==== /tmp/hubble-data/repository-size-detailed.tsv =============================

================================================================================
Finished update of repository-size.tsv (runtime: 59.870 s)
```

It looks like it runs for a while but no detailed output.

Maybe some DB changes ? 

Anyone has seen this or has any clue what the issue maybe ?

Thank
Taher.

**Comment by svasek** [comment]
Hi @tahershaakir, this issue is solved by #236.

**Comment by tahershaakir** [comment]
@svasek  Thank you very much, very helpful.  ( For now I shall just port this change to our code).

When will your PR get merged into master ?  
Whats the process/timeline like ?

**Comment by tahershaakir** [comment]
Shall be closing this as its fixed by PR #236  by Milos.
